pkg/providers/instancetype/instancetype.go

```diff
@@ -166,8 +166,8 @@ func computeRequirements(sku *skewer.SKU, vmsize *skewer.VMSizeType, architectur
 
 		// SKU capabilities
 		scheduling.NewRequirement(v1beta1.LabelSKUStorageEphemeralOSMaxSize, corev1.NodeSelectorOpDoesNotExist),
-		scheduling.NewRequirement(v1beta1.LabelSKUStoragePremiumCapable, corev1.NodeSelectorOpDoesNotExist),
```

Comment:
Just confirming, setting this here is enough to result in nodes whose SKUs have this capability getting the label correctly set, even if the user doesn't actually _specify_ this requirement, is that right?

Reply:
Yes. All single-valued requirements of instance type used are converted to labels: https://github.com/Azure/karpenter-provider-azure/blob/67902180fa6a5d01fce0f03e1d19f78d8fa2a83a/pkg/cloudprovider/cloudprovider.go#L438